scripts/uma-rewards/upunk-rewards.js

```diff
@@ -1,26 +1,54 @@
 let Web3 = require("web3")
-let web3 = new Web3('http://192.168.1.39:8545')
+let web3 = new Web3('https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161')
 let ABIS = require('./abis.js')
 const WETH = new web3.eth.Contract(ABIS.ERC20_ABI, '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2')
 const USDC = new web3.eth.Contract(ABIS.ERC20_ABI, '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48')
 let paymentPeriods = [
   {
-    startBlock: 12680434,
-    amount: 5000000000000000000000n * 82n / 100n,
-    endBlock: 12725501,
+    // 1633046400 | 01.10 | scUMA-1103
+    startBlock: 13330090, // 13412045
```

Comment:
@nateawelch once you find the time, we need confirmation to which specific end date you covered the rewards of september at, was it the last block of september or later around the 10th of october?